include/psp2/screenshot.h

```diff
@@ -1,5 +1,5 @@
 /**
- * \usergroup{SceScreenShot}
+ * \usergroup{SceScreenshot}
```

Comment:
Sorry but is that not a feet on back ? francar renamed SceScreenshot functions for a reason...